docker-compose.yml

```diff
@@ -0,0 +1,44 @@
+---
+version: "2.2"
+services:
+
+  zk:
+    image: confluentinc/cp-zookeeper:6.2.0
```

Comment:
i am wondering if we should use a different image so its similar to the kafka we have on event broker prod. plus confluent has these platform versions instead of kafka version which makes it difficult to keep track of which kafka version is underneath. but this can be an improvement as well and can be looked at a later point

Reply:
i guess we are doing this later. fine by me